hw/ip/aes/data/aes.hjson

```diff
@@ -232,6 +232,86 @@
     }
   ],
 
+  //////////////
+  // Features //
+  //////////////
+
+  features: [
+    {
+      name: "AES.KEY_LEN.128",
+      desc: "AES can perform both operations (encryption and decryption) for key length 128."
+    }
+    {
+      name: "AES.KEY_LEN.192",
+      desc: "AES can perform both operations (encryption and decryption) for key length 192."
```

Comment:
Please add a line saying that this feature can be disabled using a compile-time Verilog parameter.